sdk/src/client/node_manager/mod.rs

```diff
@@ -91,6 +91,14 @@ impl ClientInner {
         let request = request.rate_limit(&self.request_pool);
         request.await
     }
+
+    pub(crate) async fn post_request_bytes<T: DeserializeOwned>(&self, path: &str, body: &[u8]) -> Result<T> {
```

Comment:
Should we drop the json part of `post_request_json` for consistency with `get_request` ?

Reply:
done